### ISSUE TITLE
Add validation.

### ISSUE THREAD

**Issue by ghost** [orcund/TorchDrive#6]
- Validation of the model during the training must be implemented. 

- Add validation losses to training graph.

- As an additional feature model checkpointing might be done according to validation loss.
